wordpress-mobile/WordPress-iOS
# Issue/5674 different gravatar placeholders displayed in user list and user details

**Fixes** #5674 

**To test:**
1. Remove your Gravatar image
2. Go to Site -> People: Mystery man should be displayed
3. Click on your user row: On the user details page mystery man should be displayed

(You have to wait around 5 minutes between updates and tests due to caching)

Needs review: @koke
